layer/layer_data.cc

```diff
@@ -14,6 +14,8 @@
 
 #include "layer_data.h"
 
+#include <inttypes.h>
```

Comment:
nit: Could you move this below the next include to keep them sorted?

Reply:
clang-format reordered it this way (I guess because it's a c header while cstdint is a c++ header). I've reran clang-format with a different version and the space is now gone